File: Arcade/Intro/54_sumUpNumbers.js
function sumUpNumbers(inputString) {
    // Create array of numbers
    var numbers = inputString.split(new RegExp(/\D/));
    // Set sum equal to zero
    var sum = 0;
    // Loop through array
    for (let i in numbers) {
        // If the character isn't a space, add it to current number
        if (numbers[i] !== "") {
            sum += parseInt(numbers[i],10);
        }
    }
	return sum;
}

File: Arcade/Intro/C++/54_sumUpNumbers.cpp
int sumUpNumbers(std::string inputString) {
    vector<int> nums;
    string temp;
    for(char c : inputString) {
        if(c>='0'&&c<='9') {
            temp.push_back(c);
        } else {
            if(temp.size()>0) {
                nums.push_back(atoi(temp.c_str()));
                temp="";
            }
        }
    }
    if(temp.size()>0) {
        nums.push_back(atoi(temp.c_str()));
        temp="";
    }
    int sum=0;
    for(int i : nums) sum+=i;
    return sum;
}
